.changeset/warm-buckets-fold.md

```diff
@@ -0,0 +1,28 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+# Interactive
+- Introduced `<Hds::Interactive>` (a generic, "utility" component used internally by all the interactive elements like buttons and links)
+
+# Button
+- updated the button API to handle also links as `<a>`/`<LinkTo/LinkToExternal>` 
+  - it can be used in place of the `<Hds::Link/LinkTo::CTA>` component (see below)
+  - when the button is a link
+		- the text is underlined for differentiation with a normal button - ⚠️ **Visual change!**
+		- the button responds to `space` key event
+- removed the `@type` argument from the API in favour of the `type` native attribute - 🚨 **Breaking change!**
+
+#Link/LinkTo::CTA
```

Comment:
GitHub's markdown engine requires a space between the hashtag and the text to render a heading, if I remember well.
```suggestion
# Link/LinkTo::CTA
```